student.rb

```diff
@@ -1,9 +1,12 @@
 require './person'
 
 class Student < Person
+  attr_writer :classroom
+
   def initialize(age, name, parent_permission, classroom)
     super(age, name, parent_permission)
-    @classroom = classroom
+    @classroom = classroom # not needed but better safe than sorry
+    classroom.add_student(self)
   end
 
```

Comment:
- [ ] Kindly make sure to add a method for adding rentals to a person (student)

```
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```

Reply:
Done